lambdas/handlers/authoriser_handler.py

```diff
@@ -34,40 +23,42 @@
 @ensure_environment_variables(names=["SSM_PARAM_JWT_TOKEN_PUBLIC_KEY"])
 def lambda_handler(event, context):
     try:
+        logger.info(f"Authoriser handler triggered: event")
```

Comment:
Is event her meant to be `{event}`?
Also, can you move this log to be after the request_context is set?

Reply:
Ah yes, I likely left this in after debugging. Good spot!
Made the change and moved this down

Reply:
You still have event as a string there 😅

Reply:
So I do! Got it in the next commit